spec/forms/green_lanes/eligibility_form_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+
+RSpec.describe GreenLanes::EligibilityForm, type: :model do
+  describe 'validations' do
+    subject(:form) { described_class.new(params) }
+
+    before { form.valid? }
+
+    context 'when all the attributes are present' do
+      let(:params) do
+        {
+          commodity_code: '6203000000',
+          moving_goods_gb_to_ni: 'yes',
+          free_circulation_in_uk: 'yes',
+          end_consumers_in_uk: 'yes',
+          ukims: 'yes',
+        }
+      end
+
+      it { expect(form.errors).to be_empty }
+    end
+
+    context 'when an commodity_code is missing' do
```

Comment:
`when a**n** commodity_code is missing`
or it should be
`when the commodity_code is missing`

Reply:
good catch and yes the bottom i think i prefer